packages/shaku-code-annotate/src/__tests__/index.test.ts

```diff
@@ -230,6 +230,13 @@ describe("parseLine() can parse comment lines", () => {
     const result = parseLine("This is some comments from source code");
     expect(result).toBeNull();
   });
+
+  test("  */", async () => {
```

Comment:
This is not what we want, you are treating all  `       */`  as shaku-specific stuff